crates/orchestrator/src/generators/command.rs

```diff
@@ -244,12 +246,30 @@ pub fn generate_for_node(
     }
 
     // ports
+
+    // Prometheus
     tmp_args.push("--prometheus-port".into());
-    tmp_args.push(node.prometheus_port.0.to_string());
+    tmp_args.push(if options.use_default_ports_in_cmd {
+        PROMETHEUS_PORT.to_string()
+    } else {
+        node.prometheus_port.0.to_string()
+    });
```

Comment:
I think we could directly set the node.prometheus_port as a default when when initialize, and override when user set it ?